api/src/main/java/org/pmiops/workbench/api/CohortAnnotationDefinitionController.java

```diff
@@ -134,11 +132,7 @@ private CohortAnnotationDefinition findCohortAnnotationDefinition(
   }
 
   private void validateCohortExist(long cohortId) {
-    DbCohort cohort = cohortReviewService.findCohort(cohortId);
-    if (cohort == null) {
-      throw new NotFoundException(
-          String.format("Not Found: No Cohort exists for cohortId: %s", cohortId));
-    }
```

Comment:
Remove this cause it's also happening in cohortReviewService.findCohort(cohortId)